pkgs/development/libraries/malcontent/better-separation.patch

```diff
@@ -22,14 +46,13 @@ index a6c477d..f7b2f0a 100644
 +endif
  if get_option('ui').enabled()
    subdir('malcontent-control')
- endif
--subdir('pam')
++  subdir('help')
```

Comment:
This line is possibly upstreamable. There should definitely be no need to install help when ui is not installed.